src/CalendarApp.Console/Presenters/DeleteMeetingByNamePresenter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using CalendarApp.Console.Presenters.Interfaces;
+using CalendarApp.Domain.Exceptions;
+using CalendarApp.Domain.Services;
+using CalendarApp.Domain.Services.Interfaces;
+
+namespace CalendarApp.Console.Presenters;
+
+internal class DeleteMeetingByNamePresenter : IPresenter
+{
+	private readonly IMeetingsService _meetingService;
+
+	public DeleteMeetingByNamePresenter(IMeetingsService meetingService)
+	{
+		_meetingService = meetingService;
+	}
+
+	public IPresenter Action()
+	{
+		while (true)
+		{
+			try
+			{
+				var name = ReadLine();
+
+				_meetingService.DeleteMeeting(name);
+
+
+
+			}
+			catch (CalendarAppDomainException exc)
```

Comment:
it's not expected behavior for delete to throw exception when element not found
for update the same